docs/ExternalResults.asciidoc

```diff
@@ -96,7 +96,7 @@ $parser_result->tests->each(sub {
 
 my @all = $parser_result->tests->each;
 my @tests = $parser->tests->search(name => qr/1_running_upstream_tests/);
-my $first = $parser->tests->search(name => qr/1_running_upstream_tests/)->first();
+my $first = $parser->tests->find(name => qr/1_running_upstream_tests/);
```

Comment:
This is an `OpenQA::Parser::Result::OpenQA::Results` so this change is wrong.

Reply:
I don't understand. Why is it wrong?

Reply:
`$parser->tests` returns a `OpenQA::Parser::Result::OpenQA::Results` object which has nothing to do with DBIx and only has a `search` function.